cvmfs/server/cvmfs_server_util.sh

```diff
@@ -501,7 +501,15 @@ check_overlayfs_version() {
       if [ "x$scratch_fstype" = "xext3" ] || [ "x$scratch_fstype" = "xext4" ] ; then
         return 0
       fi
-      echo "overlayfs scratch /var/spool/cvmfs is type $scratch_fstype, but ext3 or ext4 required"
+      if [ "x$scratch_fstype" = "xxfs" ] ; then
+        if [ "x$(xfs_info /var/spool/cvmfs | grep ftype=1)" != "x" ] ; then
```

Comment:
Can we expect `xfs_info` to be present on the system?  Otherwise we should either use /proc/mounts or add the corresponding package dependency.

Reply:
I don't know how to get this information from /proc/mounts (I don't see a mention of ftype).

The xfs_info command is in xfsprogs, I checked with a fresh CC7 image on the CERN Openstack and the command is available by default.

Reply:
Ok, thanks for checking.  Looking again at the `if` clause, I think it will fail correctly if `xfs_info` does not exist, anyway.

Reply:
I checked, too. If the command isn't found, the proper code is returned.

Reply:
Sure but it would print an extraneous error message, right?  Should stderr be redirected to /dev/null?